bindings/cs/rl.net.native/binding_static_model.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "model_mgmt.h"
+
```

Comment:
a namespace alias might be useful for really long namespaces. For example...
```suggestion
namespace model_transport = reinforcement_learning::model_management;
```